### ISSUE TITLE
Add text export as backup

### ISSUE THREAD

**Issue by vanishakesswani** [HaikuArchives/DeskNotes#5]
There is no option to sync text so I can't take a backup . In case of a system crash I am gonna loose all my Notes .

**Comment by humdingerb** [comment]
Yes. The replicants are only saved to the tracker_shelf when Tracker quits.  If it crashes before that, all is lost. This could be improved on Tracker's side: Every added/removed replicant could trigger a rewrite of the shelf.
Nonetheless, an "Export as text" of all notes would be useful.
